src/java/org/apache/cassandra/cql3/statements/schema/CreateIndexStatement.java

```diff
@@ -221,6 +222,7 @@ Set<String> clientWarnings(KeyspacesDiff diff)
     private void validateIndexTarget(TableMetadata table, IndexMetadata.Kind kind, IndexTarget target)
     {
         ColumnMetadata column = table.getColumn(target.column);
+        AbstractType<?> baseType = column.type.unwrap();
```

Comment:
@sunil9977 this will NPE when `column` is null, that check is just on the next line after you unwrapped it.

Reply:
@smiklosovic Corrected it.